feature/search/src/test/kotlin/com/google/samples/apps/nowinandroid/feature/search/SearchViewModelTest.kt

```diff
@@ -128,4 +130,41 @@ class SearchViewModelTest {
 
         collectJob.cancel()
     }
+
+    @Test
+    fun emptySearchText_NotAddToRecentSearches() = runTest {
+        viewModel.onSearchTriggered("")
+
+        val recentSearchQueriesStream = getRecentQueryUseCase()
+        val recentSearchQueries = recentSearchQueriesStream.first()
+        val recentSearchQuery = recentSearchQueries.firstOrNull()
+
+        assertNull(recentSearchQuery)
+    }
+
+    @Test
+    fun stateIsEmptyQuery_withThreeWhiteSpacesSearchQuery() = runTest {
```

Comment:
Test names should follow the given-when-then format. So in this case it should be: 

Given a search query with 3 spaces
Then the state is empty query

Which translates to a test name of `searchTextWithThreeSpaces_isEmptyQuery`

Reply:
I'll change the test name from your suggestion.
Thank you.